esmvalcore/_task.py

```diff
@@ -593,6 +649,82 @@ def __str__(self):
         return txt
 
 
+def _get_response(url):
+    """Return information from CMIP6 Data Citation service in json format."""
+    json_data = False
+    if url.lower().startswith('https'):
+        try:
+            response = requests.get(url)
+            if response.status_code == 200:
+                json_data = response.json()
+            else:
+                logger.info('Error in the CMIP json link')
+        except IOError:
+            logger.info('Error in receiving the CMIP json file')
+    return json_data
+
+
+def _json_to_bibtex(data):
+    """Make a bibtex entry from CMIP6 Data Citation json data."""
+    url = ''.join(['https://doi.org/', data['identifier']['id']])
+    author_list = []
+    for item in data['creators']:
+        author_list.append(item['creatorName'])
+    bibtex_entry = ('@misc{' + url + ',\n\t'
+                    'url = {' + url + '},\n\t'
+                    'title = {' + data['titles'][0] + '},\n\t'
```

Comment:
What happens if any of these fields (like `data['titles']) are missing? Or does that never happen?

Reply:
@Peter9192 Thanks. I added a function that test for those fields.